docs/source/platform/operation-registry.md

```diff
@@ -386,3 +342,126 @@ By clicking on the URL listed in the `Checking for manifest changes at` message,
 If a problem occurs during the `apollo service:push` command, make sure that the running Apollo Server can be accessed from the machine where the command is being executed.
 
 Additionally, make sure that introspection is enabled on the server since introspection details are used to obtain and publish the schema.
+
+## Migrating from `0.1-alpha.4` to `0.2.0-alpha.1`
+
+### Summary of changes
+
+The operation registry has been upgraded to improve observability and robustness. These changes include: a [**stable operation manifest location**](#manifest-storage-location), registry [**metrics in the UI**](#metrics-and-usage-statistics), apollo [**client:push diagnostics**](#operation-registration-observability), and [**variant/tag awareness**](#varianttag-awareness).
+
+These update are designed to be transparent with a smooth upgrade path. To
+facilitate the upgrade, all versions of `apollo client:push` double-write to the new
+and old storage locations and the registry plugin reads from the new location
+and uses the old location as a fallback.
+
+> Note: Upgrading the apollo cli requires some extra caution if a variant/tag is defined in the `apollo.config.js` in order to ensure the proper variant/tag during push
+
+### Upgrade path
+
+While this upgrade should not require additional work, we recommend running
+the new plugin with `dryRun` and `debug` enabled. To report metrics and use
+the new storage location, upgrade:
+
+- `apollo-server` or `apollo-server-<variant>` to `2.6.3`
+- `apollo-server-plugin-operation-registry` to `0.2.0-alpha.1`.
+
+If operations have not been registered since June 6th, the plugin will
+fallback on the old location. With `debug` enabled, the plugin will log which
```

Comment:
tangential: Is there a way to pass a custom logger?

Reply:
not yet! Though that would be an interesting feature